sanity-io/structured-content-2022
# Inconsistent social logo file names & black social icons for Speaker page

# Inconsistent social logo file names & black social icons for Speaker page

## Intent

Fix the color of the white social icons appearing in Speaker pages.
![image](https://user-images.githubusercontent.com/6001131/165469596-ccaeb50d-4dae-492f-91e0-f190d3f02601.png)

Fixes [this](https://app.shortcut.com/sanity-io/story/18975/fix-color-of-social-icons-on-speaker-page) Story.

## Description

It looks like some of the logos were incorrectly named, being suffixed with `_black` despite having `fill="#FFF"`. This PR also fixes this inconsistency, and also uses SVGOMG to minify them ([because next/image does not by default](https://nextjs.org/docs/api-reference/next/image#dangerously-allow-svg)).

## Testing this PR

1. Visit [Carrie's page in the Preview Deploy](https://structured-content-2022-web-git-speaker-page-social-icons-black.sanity.build/speakers/carrie-hane) for this PR
2. Verify that the Footer's social icons remain white
3. Verify that the Speaker's social icons (under pronouns & title/company) now render as black
